src/ui/public/chrome/api/angular.js

```diff
@@ -87,12 +89,27 @@ export function initAngularApi(chrome, internals) {
           try {
             if (urlOverflow.check($location.absUrl()) <= URL_LIMIT_WARN_WITHIN) {
               toastNotifications.addWarning({
-                title: 'The URL is big and Kibana might stop working',
+                title: i18n.translate('common.ui.chrome.urlIsBigAndKibanaMightStopWorkingNotificationTitle', {
```

Comment:
I'd prefer shorter id:
```suggestion
                title: i18n.translate('common.ui.chrome.bigUrlWarningNotificationTitle', {
```